solr/core/src/java/org/apache/solr/cloud/OverseerTaskProcessor.java

```diff
@@ -189,7 +189,7 @@ public void run() {
           if (waited)
             cleanUpWorkQueue();
 
-          List<QueueEvent> heads = workQueue.peekTopN(MAX_PARALLEL_TASKS, runningZKTasks, 2000L);
```

Comment:
Thoughts on bumping up this timeout?  It reduces log spam when idle.  The method will return immediately the moment a new task item actually hits ZK.